### PR TITLE
Makefile: Fall back to curl unless bikeshed/docker, #44

### PR DESCRIPTION
**Pull request by kba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,17 +38,24 @@ channel](https://gitter.im/kba/hocr-spec).
 
 ## Building the spec
 
-To build the spec, you will need to have GNU make and either
-[bikeshed](https://github.com/tabatkins/bikeshed) or
-[docker](https://docker.com) installed.
+To build the spec, you will need to have GNU make and one of
+  * [bikeshed](https://github.com/tabatkins/bikeshed) or
+  * [docker](https://docker.com) or
+  * `curl` installed.
```
Comment (amitdo) [inline README.md]: To build the spec, you will need to have GNU make and one one of 
the following programs installed:  
- [bikeshed](https://github.com/tabatkins/bikeshed) ~~or~~
- [docker](https://docker.com) ~~or~~
- `curl`~~installed.~~

**Comment by amitdo** [comment]
Just suggesting ...

File: README.md (hunk 1 above)
Comment (kba) [reply README.md]: Makes sense, can you commit that?

Comment (amitdo) [reply README.md]: OK, but I'm leaving the pc right now, will go back in a hour or two, so when I'll come back I'll do it (unless you'll do it until then).

**Comment by kba** [comment]
I see, I thought you had already edited the README and just needed to press
save :) No hurry, thanks.

**Comment by kba** [comment]
Thank you

**Comment by amitdo** [comment]
Thank **you**

**Pull request merged**